stable/chronograf/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 name: chronograf
-version: 0.4.2
+version: 0.4.3
+appVersion: "0.4.3"
```

Comment:
should probably be 1.3, as seen in https://github.com/AdamDang/charts/blob/48692b2314e44bf3881b5e542e84977bf1101802/stable/chronograf/values.yaml#L5